skrf/networkSet.py

```diff
@@ -658,7 +658,7 @@ def cov(self, **kw) -> npy.ndarray:
         return npy.array([npy.cov(k.T) for k in smat])
 
     @property
-    def mean_s_db(self) -> 'Network':
+    def mean_s_db(self) -> Network:
```

Comment:
for my knowledge: why do you remove the quotes around Network here, and, for example, not in network.py ?

Reply:
Within the `Network` class, `Network` is not (fully) defined yet, so I'll get an error.
One could import `annotations` from `__future__`, which will become the default behaviour in Python 3.10.

As we are still supporting Python3.6 there's no `from __future__ import annotation`. This would allow to use a class name as typehint within the class. For Python < 3.7 one should just use the stringified name.

See here for a better explanation:
https://stackoverflow.com/questions/33533148/how-do-i-type-hint-a-method-with-the-type-of-the-enclosing-class

Reply:
ok, thank you for the explanation